04.android/06.solving-problems/12.Problems caused by using a Restricted Mode profile/docs.en.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Problems caused by using a Restricted Mode profile'
+published: true
+taxonomy:
+    category:
+        - docs
+---
+
+
+The owners of phones, tablets, or TVs running the Android 7+ operating system might face the problem caused by using **a profile with Restricted Mode**. If you have such a profile, **AdGuard**, like other applications that use VPN, gets restrictions on selective filtering of VPN traffic. As a result, AdGuard cannot launch protection in a **local VPN mode**. Below are described the recommendations that you can apply when a problem occurs.
```

Comment:
that you can apply when a problem occurs -> that you can apply when this problem occurs